skrub/_joiner.py

```diff
@@ -12,20 +11,23 @@
 from sklearn.utils.validation import check_is_fitted
 
 from skrub import _join_utils, _matching, _utils
-from skrub._dataframe._namespace import is_pandas, is_polars
+from skrub import _selectors as s
+from skrub._check_input import CheckInputDataFrame
+from skrub._dataframe import _common as ns
```

Comment:
```suggestion
from . import _dataframe as ns
```

also can you replace the other imports with relative ones, and I'm not too sure about the alias "ns"

Reply:
As of now we've used :
- `from ._dataframe import common as ns`
- `from . import _dataframe as sbd`

so I can put `from . import _dataframe as sbd` here